clusters/app.ci/supplemental-ci-images/telco-bastion.yaml

```diff
@@ -27,6 +27,7 @@ items:
             rm -rf /var/cache/yum/* && \
             chmod -R g+rwx /output && \
             chmod g+w /etc/passwd && \
+            mkdir /output && \
```

Comment:
Move it before this line? 

> chmod -R g+rwx /output && \

And it will work. Tested on the cluster.

Reply:
ah, my bad